cluster-provision/centos8/scripts/dnsmasq.sh

```diff
@@ -34,4 +36,4 @@ iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
 iptables -A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
 iptables -A FORWARD -i br0 -o eth0 -j ACCEPT
 
-exec dnsmasq -d ${DHCP_HOSTS} --dhcp-range=192.168.66.10,192.168.66.200,infinite
+exec dnsmasq -d ${DHCP_HOSTS} --dhcp-range=192.168.66.10,192.168.66.200,infinite --dhcp-range=::,static
```

Comment:
--dhcp-range=::
I am almost sure passing `::` to this flag means it will not allocate address

Reply:
"The optional <mode> keyword may be static which tells dnsmasq to enable 
DHCP for the network specified, but not to dynamically allocate IP 
addresses: only hosts which have static addresses given via dhcp-host or 
from /etc/ethers will be served. A static-only subnet with address all 
zeros may be used as a "catch-all" address to enable replies to all 
Information-request packets on a subnet which is provided with stateless 
DHCPv6, ie --dhcp-range=::,static"

i believe line 25 is the one that gives the ip, this is so it will answer replies,
the nodes do have the ip they should